lib/gitlab/client/licenses.rb

```diff
@@ -0,0 +1,30 @@
+class Gitlab::Client
+  # Defines methods related to licenses.
+  # @see https://docs.gitlab.com/ce/api/licenses.html
+  module Licenses
```

Comment:
Let's namespace it as `Templates`. It makes more sense considering http://docs.gitlab.com/ce/api/templates/gitignores.html and http://docs.gitlab.com/ce/api/templates/gitlab_ci_ymls.html